### ISSUE TITLE
Urosepsis

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6173]
**Preferred term label:**
Urosepsis

**Synonyms**

**Definition (free text, please give PubMed ID)**
A sepsis caused by an infection of the urinary tract (UTI).
PMID: 29493969 

**Parent term (use hpo.jax.org/app)**
Sepsis

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM: 615112

**Comment by pnrobinson** [comment]
In this case, we would use the HPO term Sepsis (https://hpo.jax.org/app/browse/term/HP:0100806), because as far as I know the phenotypic features are the same regardless of the cause of the sepsis. Of course the entire disease will be different, but I am talking about the specific phenotypic feature. HPO tries not to encode etiology.
